.github/workflows/test_branches.yml

```diff
@@ -47,20 +47,6 @@ jobs:
       uses: crate-ci/typos@master
       with: 
         config: ./.github/workflows/typos.toml
-    - name: URL Checker
```

Comment:
Should we leave this here - either as is, or as a separate job?  That way Devs would be informed about failing URLs as part of their dev process, but it wouldn't hold up the PR builds / reviews?

Reply:
I like this idea for catching issues with new URLs being added